tuf/sig.py

```diff
@@ -185,18 +185,15 @@ def get_signature_status(signable, role=None, repository_name='default',
     if valid_sig:
       if role is not None:
 
-        try:
-          # Is this an unauthorized key? (a keyid associated with 'role')
-          if keyids is None:
-            keyids = tuf.roledb.get_role_keyids(role, repository_name)
+        # Is this an unauthorized key? (a keyid associated with 'role')
+        # Note that if the role is not known, tuf.exceptions.UnknownRoleError
+        # is raised here.
+        if keyids is None:
+          keyids = tuf.roledb.get_role_keyids(role, repository_name)
 
-          if keyid not in keyids:
-            untrusted_sigs.append(keyid)
-            continue
-
-        # Unknown role, re-raise exception.
-        except tuf.exceptions.UnknownRoleError:
-          raise
+        if keyid not in keyids:
```

Comment:
Maybe I am missing something, but where is UnknownRoleError handled if it's removed here?